tools/accuracy_checker/accuracy_checker/main.py

```diff
@@ -240,6 +240,11 @@ def build_arguments_parser():
         required=False,
         type=partial(get_path, is_directory=True)
     )
+    parser.add_argument(
+        '--subsample_size', help="number of samples to be used",
+        required=False,
+        type=int
```

Comment:
```suggestion
        type=str
```
we also accept subsample ratio instead of specific number (e.g. 10% is valid value for this parameter and mean that we need to get 10% from full dataset)

Reply:
Done